rpc/ClientRPC.java

```diff
@@ -69,4 +72,12 @@ public void close() {
     public Channel channel() {
         return channel;
     }
+
+    static <RES> RES rpcCall(Supplier<RES> req) {
```

Comment:
Do o it weird that now we have to import `ClientRPC` from every class in order for those class to access `rpcCall`? What if we put the method under the common package? Especially since the method is quite generic and free of context

Reply:
That's definitely one option

Reply:
Given that it's only used by the `rpc` package I moved it to `rpc.util.RPCUtils`.